modules/ossm-config-openshift-monitoring-only.adoc

```diff
@@ -0,0 +1,99 @@
+
+:_mod-docs-content-type: PROCEDURE
+[id="ossm-config-openshift-monitoring-only_{context}"]
+= Configuring OpenShift Monitoring with Service Mesh
+:context: ossm-config-openshift-monitoring-only
+
+//Need some kind of intro
+//Possible file name may change
+//Possible assembly file may change
+//Assemblies, topic map info needs to be worked out still for 3.0, now that docs has a better idea of what is to be included for 3.0 TP1.
+
+//FIX THE STYLE THINGS ONCE DEV HAS REVIEWED/APPROVED THE CONTENT. First priority is ensuring the steps are in the correct order, and are accurate for 3.0 TP1.
+
+The following steps show how to integrate {SMProductName} with user-workload monitoring.
+
+.Prerequisite
+
+ * Monitoring for user-defined projects is enabled. See "Enabling monitoring for user-defined projects".
+
+.Procedure
+
+. Apply a `ServiceMonitor` object to monitor the Istio control plane:
++
+.Example `ServiceMonitor` object
+[source,yaml]
+----
+apiVersion: monitoring.coreos.com/v1
+kind: ServiceMonitor
+metadata:
+  name: istiod-monitor
+  namespace: istio-system
+spec:
+  targetLabels:
+  - app
+  selector:
+    matchLabels:
+      istio: pilot
+  endpoints:
+  - port: http-monitoring
+    interval: 30s
+    relabelings:
+    - action: replace
+      replacement: "basic-istio-system"
+      targetLabel: mesh_id
+----
+
+. Apply a `PodMonitor` object to collect metrics from Istio proxies:
++
+.Example PodMonitor object
+[source,yaml]
+----
+apiVersion: monitoring.coreos.com/v1
+kind: PodMonitor
+metadata:
+  name: istio-proxies-monitor
+  namespace: istio-system
+spec:
+  selector:
+    matchExpressions:
+    - key: istio-prometheus-ignore
+      operator: DoesNotExist
+  podMetricsEndpoints:
+  - path: /stats/prometheus
+    interval: 30s
+    relabelings:
+    - action: keep
+      sourceLabels: [__meta_kubernetes_pod_container_name]
+      regex: "istio-proxy"
+    - action: keep
+      sourceLabels: [__meta_kubernetes_pod_annotationpresent_prometheus_io_scrape]
+    - action: replace
+      regex: (\d+);(([A-Fa-f0-9]{1,4}::?){1,7}[A-Fa-f0-9]{1,4})
+      replacement: '[$2]:$1'
+      sourceLabels: [__meta_kubernetes_pod_annotation_prometheus_io_port,
+      __meta_kubernetes_pod_ip]
+      targetLabel: __address__
+    - action: replace
+      regex: (\d+);((([0-9]+?)(\.|$)){4})
+      replacement: $2:$1
+      sourceLabels: [__meta_kubernetes_pod_annotation_prometheus_io_port,
+      __meta_kubernetes_pod_ip]
+      targetLabel: __address__
+    - action: labeldrop
+      regex: "__meta_kubernetes_pod_label_(.+)"
+    - sourceLabels: [__meta_kubernetes_namespace]
+      action: replace
+      targetLabel: namespace
+    - sourceLabels: [__meta_kubernetes_pod_name]
+      action: replace
+      targetLabel: pod_name
+    - action: replace
+      replacement: "basic-istio-system"
+      targetLabel: mesh_id
+----
++
+. Go to **Observe** -> **Metrics** and try the query `istio_requests_total`.
```

Comment:
These instructions are the same as the 2.X version so take this comment with a grain of salt but it might be worth noting here that it can take awhile for the new endpoints to get scraped by prometheus and because of that it can take awhile for the `istio_requests_total` query to return anything. I don't want to over-complicate this since this is what we already had for 2.X but there are also other ways of verifying that 1. Your app endpoints are being scraped correctly 2. Your controlplane metrics are being scraped correctly. I'm fine with merging this as is though since this is what was already there for 2.X.